bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -956,7 +956,11 @@ export default function Whiteboard(props) {
 
   if (currentTool && !isPanning && !tldrawAPI?.isForcePanning) tldrawAPI?.selectTool(currentTool);
 
-  if (backgroundShape?.src && backgroundShape?.complete && backgroundShape?.src !== bgShape?.src) {
+  if (backgroundShape?.src // if there is a background image

```

Comment:
```
  if (backgroundShape?.src // if there is a background image
```

Technically we checked for `backgroundShape?` already and could remove the next 3 checks for the same. If you'd like to make sure we never drop the `?` we could even do it as:

`if ( backgroundShape && backgroundShape.src // if there is a background image` and then drop the `?` from next 3 lines